### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by zoe-stockholm**
One is for duplicated posts display in LatestPostsPlugin.
Another one is for the author selector in admin for AuthorPlugin.

### CODE REVIEW

**Comment by smaaland** [comment]
This breaks, because if an author have say 20 published posts, his name will show up in the admin list 20 times. Also it makes it possible to choose an author who has not yes published anything but will do in the future without having to change the plugin instance. The we can override the template instead to include a conditional on whether or not to show authors that have not published anything yet.

**Comment by smaaland** [comment]
For instance like so:

```
{% for author in authors_list %}
	    {% if author.get_full_name %}
		    {% if author.count > 0 %}
		    <li><a href="{% url 'djangocms_blog:posts-author' author.get_username %}">
		      {{ author.get_full_name }}
		      <span>(
		        {% if author.count > 0 %}
		        {% blocktrans count articles=author.count %}1 news story{% plural %}{{ articles }} news stories{% endblocktrans %}
		        {% else %}{% trans "0 news stories" %}{% endif %}
		        )</span>
		      </a>
		     </li>
		     {% endif %}
	     {% endif %}
	      {% empty %}
	      <li class="blog-empty"><p>{% trans "No news stories found." %}</p></li>
      {% endfor %}
```

File: djangocms_blog/forms.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class LatestEntriesForm(forms.ModelForm):
 
     def __init__(self, *args, **kwargs):
         super(LatestEntriesForm, self).__init__(*args, **kwargs)
-        self.fields['tags'].widget = TagAutoSuggest('taggit.Tag')
+        # self.fields['tags'].widget = TagAutoSuggest('taggit.Tag')
```
Comment (yakky) [inline djangocms_blog/forms.py]: Why removing tags here?

File: djangocms_blog/models.py [hunk 2]
```diff
@@ -259,13 +259,14 @@ def get_posts(self, request):
...
             posts = posts.filter(categories__in=list(self.categories.all()))
+        posts = posts.distinct()
         return posts[:self.latest_posts]
 
 
 class AuthorEntriesPlugin(BasePostPlugin):
     authors = models.ManyToManyField(
         dj_settings.AUTH_USER_MODEL, verbose_name=_('Authors'),
-        limit_choices_to={'djangocms_blog_post_author__publish': True}
+        # limit_choices_to={'djangocms_blog_post_author__publish': True}
```
Comment (yakky) [inline djangocms_blog/models.py]: The suppression of this filter should be at least managed by a setting

**Comment by yakky** [comment]
@zoe-stockholm @smaaland still interested in having this in?

**Pull request closed without merging**